### PR TITLE
Add Julia Data Science

### PR DESCRIPTION
**Pull request by rikhuijzer**
## What does this PR do?

Add the Julia Data Science book

### Description

This is an open source and open access book on how to do Data Science using the Julia programming language. Our target audience are PhD candidates from all fields of applied sciences. Industry analysts, upcoming and established researchers are also welcome as readers.

### Why is this valuable (or not)?

We aim for the book to be a more coherent introduction into Julia than just going into the documentation of the respective packages. This is valuable for people who just start using the language. Also, the link will remain valuable because the book and PDF are build via CI and lots of efforts are invested to automate as much of the building as possible in order to make maintenance easy in the future.

### How do we know it's really free?

The license is visible in the linked website and in the [GitHub repo](https://github.com/JuliaDataScience/JuliaDataScience).

### For book lists, is it a book? For course lists, is it a course? etc.

Yes, it is a book.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Followup

- Check the output of Travis-CI for linter errors!

### CODE REVIEW

File: books/free-programming-books.md [hunk 1]
```diff
@@ -1988,6 +1988,7 @@ Kerridge (PDF) (email address *requested*, not required)
 
 * [Introducing Julia](https://en.wikibooks.org/wiki/Introducing_Julia) - Wikibooks
 * [Julia by Example](http://samuelcolvin.github.io/JuliaByExample) - Samuel Colvin (GitHub repo)
+* [Julia Data Science](https://juliadatascience.io) - Jose Storopoli, Rik Huijzer, and Lazaro Alonso
```
Comment (davorpa) [inline books/free-programming-books.md:1991]:
```suggestion
* [Julia Data Science](https://juliadatascience.io) - Jose Storopoli, Rik Huijzer, and Lazaro Alonso (HTML, [PDF](https://juliadatascience.io/juliadatascience.pdf))
```
Add book formats

**Review by davorpa — CHANGES_REQUESTED** [review]
Hi Rick

Thanks for your book.

Could you accept suggestions to register book formats?

**Comment by eshellman** [comment]
Excellent addition. No need to add a link to the pdf, but davorpa's suggestion is not a bad idea.
Thanks for making your book free! As it has an Open License, consider adding it to Unglue.it and Archive.org.
Welcome to free-programming-books!

**Comment by storopoli** [comment]
Thanks for the PR folks!

**Pull request merged**